astropy/nddata/ccddata.py

```diff
@@ -129,6 +129,9 @@ class CCDData(NDDataArray):
             If the unit is ``None`` or not otherwise specified it will raise a
             ``ValueError``
 
+    psf : `numpy.ndarray` or None, optional
+        Normalized PSF image at the center of this image.
```

Comment:
```suggestion
        Normalized image (2D array) representing the PSF at the center of this image.
```
I'd be a bit more explicit here (if I understood correctly from the PR description); my first thought when reading this was "does this mean the 'PSF image' needs to be centred in some way?"
Without knowing much about how this will be typically used or how it is implemented in the Rubin data, might also be helpful to comment on the above point – i.e. would the PSF still be centred from that input, or could it in fact be offset to produce an image shift? But from the added tests it looks as if the implementation as such does not make particular assumptions on the shape of the PSF, so perhaps that is a bit too specific.

Reply:
I didn't want to make any assumptions in the code about how a user would implement the PSF image for their specific purposes. I could see some details about that normalization being different for e.g. a radio or x-ray image vs. one from LSST, and I don't think we want to encode any assumptions in the code here. I'll take your docstring suggestion, but remove the `2d array`, since ccddata doesn't assume that `data` is 2d either.

We'll want to have some docs about how to e.g. shift this PSF image around the actual image to use it in measurements, but I think that's out of scope for this PR (we want to be able to start making sample alert packets for LSST, but exactly how they'll be used with astropy code is up in the air).

Reply:
How about just removing the word "nromalized"? You say [here](https://github.com/astropy/astropy/pull/13743/files#diff-8af8355c2d4da08d8c5f4f2faf410fa23c3d0e4a17f7981c284cd4c3fefa8873R142) that `NDData/CCDdata` don't do anything the PSD except carry it around, so there is not a requirement for normalization.

Reply:
@mwcraig : I guess we could take the word `normalized` out from here, and add something in the narrative docs (or maybe to a parenthetical here?) to the effect of "the PSF image is typically normalized to sum to unity, to ensure that convolving it with the image is a flux-preserving operation"?